service/src/main/resources/db/migration/V509__nekku_product.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TYPE public.nekku_product_meal_time AS ENUM ('BREAKFAST', 'LUNCH', 'SNACK', 'DINNER', 'SUPPER');
+
+CREATE TYPE public.nekku_product_meal_type AS ENUM ('VEGAN', 'VEGETABLE');
```

Comment:
mistä nää mahdolliset arvot tulee? swaggerissä en huomannut tällaista rajausta

Reply:
ja on varmaa ettei nekun vastaus koskaan sisällä mitään muita arvoja? koska silloin koko synkkaus failaa heti

Reply:
Näin se kyllä on. Voin varmistaa tästä vielä, mutta tämänhetken tiedon mukaan muita arvoja ei ole.

Reply:
Sain vastauksen Nekku-järjestelmästä ja näitä arvoja ei pysty siellä kovinkaan helposti muuttamaan ja sanoi, että tehdään sitten koodimuutoksia, jos tarvitsee jotain tehdä. 

Olisiko nämä katselmoinnin läpikäynnit ok tämän PRn osalta ok?